### ISSUE TITLE
GetPubMedByID fails with a vector of PMIDs larger than 17 

### ISSUE THREAD

**Issue by dalewsteele** [ropensci/RefManageR#85]
When provided a character vector of 18 or more PMIDs, GetPubMedByID fails with the error below:

>  pm <- GetPubMedByID(pmid[1:18], db="pubmed")
```
Error in vapply(unclass(x), attr, "", "key") : values must be length 1,
 but FUN(X[[17]]) result is length 2
```
Thanks.

**Comment by mwmclean** [comment]
I need the contents of `pmid[1:18]` to assist you with this. This is almost certainly an issue with parsing the result returned from PubMed for one of the IDs you are working with. For example, `GetPubMedByID(rep(c("11209037", "21245076"), each = 9))` works fine on my machine.

**Comment by dalewsteele** [comment]
Apologies for not including a reproducible example.  See the 18 PMIDs below.

```
pmid <- dput(pmid)
c("32595783", "31923797", "32175752", "32960519", "32873500", 
"33364320", "32070417", "32404390", "32207667", "31697698", "32452818", 
"32022827", "31927162", "32087839", "33309538", "32401225", "32189096", 
"32011840")
```
pm1 fails with: "Error in vapply(unclass(x), attr, "", "key") : values must be length 1,
 but FUN(X[[17]]) result is length 2"
pm2 and pm3 succeed.
```
BibOptions(check.entries = FALSE)
pm1 <- GetPubMedByID(pmid, db="pubmed")
pm2 <- GetPubMedByID(pmid[1:17], db="pubmed")
pm3 <- GetPubMedByID(pmid[18], db="pubmed")

```

**Comment by mwmclean** [comment]
Hi, this was fixed in 3337cbe if you can download and install the latest version of the package. Thanks for reporting.
```
devtools::install_github("ropensci/RefManageR")
```
